api-catalog-ui/frontend/src/components/ServiceTab/InstanceInfo.jsx

```diff
@@ -22,14 +22,15 @@ export default class InstanceInfo extends Component {
         if (apiInfo !== null) {
             apiId = apiInfo.apiId;
         }
+        const { hideServiceInfo } = selectedService;
         return (
             <Shield title="Cannot display information about selected instance">
                 <div className="apiInfo-item">
                     <Tooltip key={selectedService.baseUrl} title="The instance URL for this service" placement="bottom">
                         <Typography>
                             {/* eslint-disable-next-line jsx-a11y/label-has-for */}
                             <label htmlFor="instanceUrl">Instance URL:</label>
-                            <span id="instanceUrl">{selectedService.baseUrl}</span>
+                            {!hideServiceInfo && <span id="instanceUrl">{selectedService.baseUrl}</span>}
```

Comment:
could you please add some test? What if `selectedService` does not have the property `hideServiceInfo`?

Reply:
Thanks for noticing, I will add the check and a test